slsim/Sources/SupernovaeCatalog/supernovae_sample.py

```diff
@@ -0,0 +1,106 @@
+from astropy.table import Table, hstack
+from slsim.Pipelines.skypy_pipeline import SkyPyPipeline
+from slsim.Sources import random_supernovae
+import numpy as np
+
+
+class SupernovaeCatalog(object):
+    """Class to generate a supernovae catalog."""
+
+    def __init__(
+        self,
+        sn_type,
+        band,
```

Comment:
is it possible to have the class of the SNe for multiple bands at once? If we do multi-band time series, I see it as a challenge to generate multiple of these classes with exactly the same configurations